sites/public/src/lib/helpers.tsx

```diff
@@ -144,6 +148,7 @@ export const getListings = (listings) => {
             : undefined,
           statuses: [getListingApplicationStatus(listing)],
           description: listing.name,
+          fallbackImageUrl: IMAGE_FALLBACK_URL,
```

Comment:
This isn't working for me on the listings page. The fallback image is showing up on the details page, but on the listings page it is just a gray image 
![image](https://github.com/bloom-housing/bloom/assets/42942267/01e7df63-ae86-41a8-abd1-02435afac870)

Reply:
I assume that you used empty string`""` as image path, which will result in that gray `div`. I changed `photos.ts` to return fallback if no image in listing available (which shouldn't happen, as it's required in core). So now i don't see a way to have empty string passed to `images` prop in `ImageCard`.
Let me know if that was the case, if not I would need props you've passed to `images`